### DISCUSSION TITLE
PHP 8 comaptibility

### DISCUSSION THREAD

**Discussion by difreo** [bootscore/bootscore#800] [Q&A]
Hello, when will the theme php 8 compatible? Now running on php 8 throws compile errors...

**Comment by crftwrk** [comment]
Hi,

theme is ready for PHP 8 since a long time. If compiler throws an error, it should be something wrong in your SCSS. It would be awesome helpful to get more info about what error is thrown, what the error says.

Check here https://bootscore.me/documentation/theme/css-scss-compiler/#errors
